moveit_core/CMakeLists.txt

```diff
@@ -51,8 +51,14 @@ else()
 endif()
 
 find_package(octomap REQUIRED)
+
 find_package(ruckig REQUIRED)
+# work around catkin_package not fetching the interface includes from the target
+# to forward to downstream dependencies. These includes do not have to be added
+# globally in this package because the target is correctly imported here.
```

Comment:
I don't understand what you mean by "add globally" here. What about this simplification:
```suggestion
# Set ruckig_INCLUDE_DIRS and ruckig_LIBRARIES for downstream packages
```

Reply:
I changed the string a bit. I wanted a note in the file explaining why `include_directories` below lists all other external packages, but not this one.